redaxo/src/core/lib/system_report.php

```diff
@@ -54,18 +54,23 @@ public function get()
                 continue;
             }
 
-            $sql = rex_sql::factory($dbId);
-
             $dbData = [];
-            $dbData['Version'] = $sql->getDbType().' '.$sql->getDbVersion();
 
-            if (1 === $dbId) {
-                $dbData['Character set'] = rex::getConfig('utf8mb4') ? 'utf8mb4' : 'utf8';
-            }
+            try {
+                $sql = rex_sql::factory($dbId);
 
-            $security = rex_setup::checkDbSecurity();
-            if ($security) {
-                $dbData['Warning'] = implode('<br/>', $security);
+                $dbData['Version'] = $sql->getDbType().' '.$sql->getDbVersion();
+
+                if (1 === $dbId) {
+                    $dbData['Character set'] = rex::getConfig('utf8mb4') ? 'utf8mb4' : 'utf8';
+
+                    $security = rex_setup::checkDbSecurity();
+                    if ($security) {
+                        $dbData['Warning'] = implode('<br/>', $security);
```

Comment:
Diese Warnung wird nur für die DB 1 gecheckt, deswegen habe ich sie in das if verschoben. Vorher wurde die Warnung für DB 1 auch bei den weiteren DBs angezeigt.